network/bridge_networkclient_linux.go

```diff
@@ -46,6 +48,43 @@ func (client *LinuxBridgeClient) CreateBridge() error {
 	return epcommon.DisableRAForInterface(client.bridgeName)
 }
 
+func (client *LinuxBridgeClient) AddRoutes(nwInfo *NetworkInfo, interfaceName string) error {
```

Comment:
imo, the method signature of interface should be like this: AddRoutes(Routes []RouteInfo).

can add a field in nwinfo structure which takes []RouteInfo. The routes should be populated in nwInfo structure in cni/network/network.go: and this function should iterate the RouteInfo array and call netlink addiproute. 

we should not be comparing ipamtype in this function